.github/workflows/check-file-updates.yaml

```diff
@@ -0,0 +1,65 @@
+name: Check config and readme updates
+on:
+  pull_request:
+    paths:
+      - "apis/**"
+jobs:
+  file-updates:
+    permissions: write-all
+    name: Test file updates
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run make commands
+        run: |
+          make generate && make manifests && make api-docs
+      - name: Check git status and find new files
+        id: git_status
+        run : |
+          if [[ -n $(git status -s) ]]
+          then
+            echo "New files to commit available"
+            echo "is_files_to_commit_present=true" >> $GITHUB_OUTPUT
+            delim=""
+            config_files_joined=""
+            files=$(git status --porcelain | egrep "apis|config" | cut -b4-)
+            for file in $files; do
+              config_files_joined="$config_files_joined$delim$file"
+              delim=","
+            done
+            echo "config_files=$config_files_joined" >> $GITHUB_OUTPUT
+            delim=""
+            docs_files_joined=""
+            files=$(git status --porcelain | egrep "docs" | cut -b4-)
+            for file in $files; do
+              docs_files_joined="$docs_files_joined$delim$file"
+              delim=","
+            done
```

Comment:
these are just the same function but on different matching pattern?

Reply:
ah yes, I guess they can be extracted to a separate function

Reply:
I guess this part can be removed, and we can just use a git status to list the files 😅